protocol/x/clob/keeper/process_single_match.go

```diff
@@ -469,8 +468,8 @@ func (k Keeper) setOrderFillAmountsAndPruning(
 	newTotalFillAmount satypes.BaseQuantums,
 	curPruneableBlockHeight uint32,
 ) *types.OffchainUpdates {
-	// Note that stateful orders are never pruned by `BlockHeight`, so we set the value to `MaxUInt32` here.
-	pruneableBlockHeight := uint32(math.MaxUint32)
+	// Note that stateful orders are never pruned by `BlockHeight`, so we set the value to `0` here.
+	pruneableBlockHeight := uint32(0)
```

Comment:
Setting to 0 here should save us 4 bytes / stateful order

Reply:
Are you sure this is a safe assumption to make? Will anything panic if there exists something in state which "should" have already been pruned?